src/error.ts

```diff
@@ -58,3 +58,14 @@ export class BSONVersionError extends BSONError {
     );
   }
 }
+
+/** @public */
```

Comment:
```suggestion
/**
 * An error generated when BSON functions encounter an unexpected input
 * or reaches an unexpected/invalid internal state
 *
 * @public
 * @category Error
 */
```

And can we add ` * @category Error` to the other errors while we're here, a little docs++